File: ControlRemote/Web/ClientApp/src/app/models/ActionSortByUserLoginModel.ts
import { ActionSortByDateTimeModel } from "./ActionSortByDateTimeModel";

export class ActionSortByUserLoginModel {
    userDomain: string;
    userLogin: string;
    name: string;
    station: string;
    commands: ActionSortByDateTimeModel[];

    public constructor(_userDomain: string, _userLogin: string, _name: string, _station: string, _commands: ActionSortByDateTimeModel[]) {
        this.userDomain = _userDomain;
        this.userLogin = _userLogin;
        this.name = _name;
        this.station = _station;
        this.commands = _commands;
    }
}

File: ControlRemote/Web/Dto/ActionSortByUserLoginModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class ActionSortByUserLoginModel
    {
        public string UserLogin { get; set; }
        public string Station { get; set; }
        public List<ActionSortByDateTimeModel> Commands { get; set; }
    }
}
